spacelift/resource_mounted_file.go

```diff
@@ -73,6 +73,12 @@ func resourceMountedFile() *schema.Resource {
 				Optional:    true,
 				ForceNew:    true,
 			},
+			"description": {
+				Type:        schema.TypeString,
+				Description: "Free-form description of the mounted file",
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
Should this really trigger resource recreate?
```suggestion
				ForceNew:    false,
```

Reply:
yes, it should. we don't have an "update resource" method implemented here, and I would need to implement one just for this. the implementation of it isn't super obvious.


Also, this follows the same pattern as `resource_environment_variable`. it's not ideal, but it'll work.

https://github.com/spacelift-io/terraform-provider-spacelift/blob/dee0f609998116bc85863c84c7d68bc88015e5a7/spacelift/resource_environment_variable.go#L85-L90

Reply:
Got it, thanks for the context. I just wanted to be sure it was not a copy pasta error 🙏🏻

Reply:
thanks for noticing anyway!